src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
OK, I'm fine with this change as well, I just don't want to go round in circles. Because this PR goes in the same direction as currently code has, I'd like to land it as-is. Removing all variable modules removal of which decreases the built size can be done separately later.

If there are no other remarks to this PR, I'd like to land it & close other ones.